QUEST_LV_0100.tsv

```diff
@@ -556,12 +556,12 @@ QUEST_LV_0100_20150317_000551	How many things are hiding in this canyon?{nl}More
 QUEST_LV_0100_20150317_000552	According to the old records, the king erased the stonemason's memories using magic to protect the Mausoleum.{nl}I think it's amazing that he erased only those memories.
 QUEST_LV_0100_20150317_000553	Liaison Official Nian
 QUEST_LV_0100_20150317_000554	You saw that, right? That's why I told you not to touch it.{nl}Because of this, we received an order to withdraw from this place.{nl}
-QUEST_LV_0100_20150317_000555	Well, if I am to be involved in this, can you at least do me a favor?{nl}Tell Adelle at the Research Camp to watch out for these kind of stones.
+QUEST_LV_0100_20150317_000555	Well, if I am to be involved in this, can you at least do me a favor?{nl}Tell Adelle at the Excavation Team Camp to watch out for these kind of stones.
```

Comment:
Yeah, on this map there's no Research Camp, and the quest icon is at the Excavation Team Camp.